src/test/php/fix-php-compatibility.php

```diff
@@ -123,6 +123,69 @@
             "#[\\ReturnTypeWillChange]\npublic function hasChildren",
         ),
     ),
+    __DIR__ . '/../../../vendor/phpunit/php-code-coverage/src/Report/Html/Renderer/File.php' => PHP_VERSION >= 7 ? array(
+        array(
+            '$numTests = count($coverageData[$i]);',
+            '$numTests = count($coverageData[$i] ?? []);',
+        ),
+    ) : array(),
+    __DIR__ . '/../../../vendor/phpunit/phpunit/src/Extensions/PhptTestCase.php' => array(
+        array(
+            'public function count()',
+            "#[\\ReturnTypeWillChange]\npublic function count()",
+            '#[\\ReturnTypeWillChange]',
+        ),
+        array(
+            'xdebug.default_enable=0',
+            'xdebug.mode=coverage',
+            'xdebug.mode=coverage',
+        ),
+    ),
+    __DIR__ . '/../../../vendor/phpunit/phpunit/src/Framework/TestCase.php' => array(
+        array(
+            'public function count(',
+            "#[\\ReturnTypeWillChange]\npublic function count(",
+            '#[\\ReturnTypeWillChange]',
+        ),
+    ),
+    __DIR__ . '/../../../vendor/phpunit/phpunit/src/Framework/TestSuite.php' => array(
+        array(
+            'public function count(',
+            "#[\\ReturnTypeWillChange]\npublic function count(",
+            '#[\\ReturnTypeWillChange]',
+        ),
+        array(
+            'public function getIterator(',
+            "#[\\ReturnTypeWillChange]\npublic function getIterator(",
+            '#[\\ReturnTypeWillChange]',
+        ),
+    ),
+    __DIR__ . '/../../../vendor/phpunit/phpunit/src/Framework/TestResult.php' => array(
+        array(
+            'public function count(',
+            "#[\\ReturnTypeWillChange]\npublic function count(",
+            '#[\\ReturnTypeWillChange]',
+        ),
+        array(
+            'public function getIterator(',
+            "#[\\ReturnTypeWillChange]\npublic function getIterator(",
+            '#[\\ReturnTypeWillChange]',
+        ),
+    ),
+    __DIR__ . '/../../../vendor/phpunit/php-code-coverage/src/CodeCoverage.php' => PHP_VERSION >= 7 ? array(
```

Comment:
Brackets make it more readable.

```suggestion
    __DIR__ . '/../../../vendor/phpunit/php-code-coverage/src/CodeCoverage.php' => (PHP_VERSION >= 7) ? array(
```